docs/src/main/asciidoc/index.adoc

```diff
@@ -18,6 +21,25 @@ applications for the server
 * _Model Reference_ the WildFly Model Reference provides information about all subsystem
 configuration options generated directly from the management model.
 
+* _Security Guide_ for those wanting to understand how to secure WildFly server and applications.
+
+* _Client Guide_ for those wanting to understand WildFly client.
+
+* _Quickstarts_ for those wanting to jump on code and start using WildFly.
+
+[[installation-guides]]
+== Installation Guides
+
+* Zipped WildFly installation, WildFly server provisioned with Galleon or WildFly Bootable JAR, the 
+link:Installation_Guide{outfilesuffix}[Installation Guide] 
```

Comment:
In all other sections listed the link is almost at the start of the bullet point making it appear to the left, this is very deep into the sentence meaning it can appear far to the right.

Reply:
Thanks, I aligned with others.

Reply:
@jfclere I suggest this becomes: 

The Installation Guide help you identify the kind of WildFly installation that best fits your applications deployments needs: a WildFly zip installation, a WildFly server provisioned with Galleon or a WildFly Bootable JAR.

The other 2 bullets should start with "The " before the link, so this structure means the 3 look consistent.

I say "a WildFly zip installation" because 'installation' can mean the result of installing or the process of installing. "Zipped WildFly installation" is not correct for the result of installing, as the result isn't zipped. (But bootable jar is!)

Reply:
Ok, followed your suggestion.